src/Drivers/Azki/Azki.php

```diff
@@ -77,8 +77,8 @@ public function purchase()
         if (empty($details['phone']) && empty($details['mobile'])) {
             throw new PurchaseFailedException('Phone number is required');
         }
-        if (count($this->getItems()) == 0) {
-            throw new PurchaseFailedException('Items is required');
+        if (!isset($details['items']) ||  count($details['items']) == 0) {
```

Comment:
Are you sure that `items` field is mandatory? please add a link to the corresponding documents.

Reply:
@khanzadimahdi  
https://api-docs.azkivam.com/
https://api-docs.azkivam.com/api.html?#api-Default-createTicket

also when sum of price items not equal main price, return error  ` "32" => "Invalid Invoice Data",`